src/osx/webview_webkit.mm

```diff
@@ -740,41 +673,47 @@ - (id)initWithWxWindow: (wxWebViewWebKit*)inWindow
     return self;
 }
 
-- (void)webView:(WebView *)sender
-    didStartProvisionalLoadForFrame:(WebFrame *)frame
+- (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(NSDictionary*)change context:(void *)context
 {
-    webKitWindow->m_busy = true;
+    if (context == webKitWindow)
+    {
+        wxWebViewEvent event(wxEVT_WEBVIEW_TITLE_CHANGED,
+                             webKitWindow->GetId(),
+                             webKitWindow->GetCurrentURL(),
+                             "");
+
+        event.SetString(webKitWindow->GetCurrentTitle());
+
+        if (webKitWindow && webKitWindow->GetEventHandler())
+            webKitWindow->GetEventHandler()->ProcessEvent(event);
```

Comment:
Hmm, is it really possible to not have an event handler? Also, testing for `webKitWindow` here is surely redundant because we've just dereferenced it above. So I'd just do
```suggestion
        webKitWindow->ProcessWindowEvent(event);
```
here (and in several other places below).

Reply:
No objection on my part, just copy&paste from previous implementation